docs/reference/modules/asset.md

```diff
@@ -12,15 +12,22 @@ The `asset` module serves to organize, process, and output all project JavaScrip
 
 ## Options
 
-|  Property | Type | Description |
-|---|---|---|
-| [`refreshOnRestart`](#refreshonrestart) | Boolean | If set to `true`, the browser will refresh on Apostrophe app restart. |
-| [`watch`](#watch) | Boolean | If set to `false`, none of the UI assets will be watched to trigger a restart. |
-| `watchDebounceMs` | Integer | Time in milliseconds to wait before re-triggering a restart on asset change. |
-| [`uploadfs`](#uploadfs) | Object | Can be used to configure an `uploadfs` instance. |
-| [`rebundleModules`](#rebundlemodules) | Object | Used to direct project wide asset files into new bundles. |
-| [`devSourceMap`](#devsourcemap) | String or `false` | Overrides the `devtool` setting of `webpack` for the admin UI build. |
-| [`breakpointPreviewMode`](#breakpointPreviewMode) | object | Enables and sets screen sizes for mobile preview. |
+|  Property | Type | Description | Configuration Location |
+|---|---|---|---|
+| [`refreshOnRestart`](#refreshonrestart) | Boolean | If set to `true`, the browser will refresh on Apostrophe app restart. | Both `app.js` and module `index.js` |
+| [`watch`](#watch) | Boolean | If set to `false`, none of the UI assets will be watched to trigger a restart. | Both `app.js` and module `index.js` |
+| `watchDebounceMs` | Integer | Time in milliseconds to wait before re-triggering a restart on asset change. | Both `app.js` and module index |
+| [`uploadfs`](#uploadfs) | Object | Can be used to configure an `uploadfs` instance. | Both `app.js` and module `index.js` |
+| [devSourceMap](#devsourcemap) | String or false | Overrides the devtool setting of webpack for the admin UI build. | Both `app.js` and module `index.js` |
+| [`rebundleModules`](#rebundlemodules) | Object | Used to direct project wide asset files into new bundles. | Both `app.js` and module `index.js` |
+| [`breakpointPreviewMode`](#breakpointPreviewMode) | object | Enables and sets screen sizes for mobile preview. | Both `app.js` and module `index.js` |
+| [`hmr`](#hmr) | String or Boolean | Controls Hot Module Replacement mode. Values: `'public'` (default), `'apos'`, or `false` | `app.js` only |
+| [`hmrPort`](#hmrport) | Number | Sets custom WebSocket server port for HMR. Defaults to ApostropheCMS server port. | `app.js` only |
+| [`productionSourceMaps`](#productionsourcemaps) | Boolean | If `true`, includes source maps in production builds. | `app.js` only |
+
+::: info
+The `devSourceMap` option has been removed as it was specific to webpack configuration. Source map behavior is now controlled through the `productionSourceMaps` option for Vite builds.
```

Comment:
Vite always does sourcemaps in development (I wasn't able to switch them off in fact).

Reply:
I thought I had already removed that tip.